### ISSUE TITLE
Onboarding Request: Lisa's fake tool

### ISSUE THREAD

**Issue by lpfeif** [noaa-fisheries-integrated-toolbox/onboard-and-update#9]
### Active Development

Yes

### Tool name

Lisa's Fake Tool

### Tool abbreviation

LFT

### Author(s)

Lisa Pfeiffer

### NOAA Internal

Yes

### Maintainer Name

Lisa Pfeiffer

### Maintainer Email

lisa.pfeiffer@noaa.gov

### Background Text

This tool writes the "results" sections of my papers, which is my most hated section. 

### Citation

website

### User Organizations

NOAA NWFSC

### Toolbox Drawers

- [ ] Ecosystems
- [ ] Fish and Fisheries
- [ ] Human Dimensions
- [ ] Protected Species
- [X] General Modeling and Productivity Tools

### Keywords

- [ ] ADMB
- [ ] Age Structured Model
- [ ] Bayesian Modeling
- [ ] Bycatch
- [ ] C++
- [ ] Calculator
- [ ] Data Exploration
- [ ] Data Limited Method
- [ ] Data Visualization
- [ ] Decision Support Tool
- [ ] Developer Utilities
- [X] Economics
- [ ] Ecosystem
- [ ] Ensemble Modeling
- [ ] Fish Biology
- [ ] Fisheries
- [ ] Gradient Solver
- [ ] Guided User Interface
- [ ] Individual Based Model
- [ ] Legacy Tool
- [ ] Length Structured Model
- [ ] Management Strategy Evaluation
- [ ] Marine Mammals
- [ ] Maximum Likelihood Estimation
- [ ] Model Builder
- [ ] Model Comparision
- [ ] Multispecies Model
- [ ] Natural Mortality Model
- [ ] NOAA Branding
- [ ] North Pacific
- [ ] Output Visualization
- [ ] Projection Modeling
- [ ] R Package
- [ ] Reference Points Model
- [ ] Risk Assessment Tool
- [ ] Salmon
- [ ] Sampling Decision Tool
- [ ] Shiny
- [ ] Simulation
- [ ] Size Spectra Model
- [ ] Size Structured Model
- [ ] Socioeconomics
- [ ] Spatial Model
- [ ] Species Distribution Model
- [ ] Standardized Reporting
- [ ] Stock Assessment Model
- [ ] Stock Assessment Projections
- [ ] Stock Assessment Tool
- [ ] Surplus Production Model
- [ ] Tag Data Model
- [ ] Teaching Tool
- [ ] Time Series
- [ ] TMB
- [ ] User Utilities
- [ ] Virtual Population Analysis
- [ ] Visualization
- [ ] Web App

### Uses Github Releases

I plan to use them in the near future

### Code of Conduct

- [X] I agree to follow the FIT's [Code of Conduct](https://github.com/nmfs-fish-tools/Resources/blob/main/CODE_OF_CONDUCT.md)

**Comment by lpfeif** [comment]
Citation is a required link--what if it's not yet published? 
"Executable Link" --is it bad that I don't know what this means?
Software badges--I also don't know what badges are

**Comment by k-doering-NOAA** [comment]
Automatically generated JSON for the onboarding request: 
```
{
  "active_development": true,
  "tool_name": "Lisa's Fake Tool",
  "tool_abbreviation": "LFT",
  "authors": "Lisa Pfeiffer",
  "noaa_internal": true,
  "maintainer_name": "Lisa Pfeiffer",
  "maintainer_email": "lisa.pfeiffer@noaa.gov",
  "background_text": "This tool writes the \"results\" sections of my papers, which is my most hated section. ",
  "citation": "website",
  "user_organizations": ["NOAA NWFSC"],
  "toolbox_drawers": ["General Modeling and Productivity Tools"],
  "keywords": ["Economics"],
  "uses_github_releases": false
}
```

**Comment by k-doering-NOAA** [comment]
thanks @lpfeif for the comments, I think illustrates that those form components need more explanation!

**Comment by k-doering-NOAA** [comment]
thanks @lpfeif for the comments! I tried to address your comments, but may still need your help in addressing them fully.

> Citation is a required link--what if it's not yet published?

I went back and forth on whether it should be required, but then decided that I don't want people just to skip citation since it is a commonly asked question. I added in a sentence to the form about using [Zenodo](https://zenodo.org/) to get a DOI for source code. I think it is pretty easy to use, so I would like to see us use this more!

> "Executable Link" --is it bad that I don't know what this means?

Does this description help clarify = `Link the compiled version of the tool, if available.`? If not, let me know so I can make this description better.

> Software badges--I also don't know what badges are

I added this sentence to the form "Badges are used to indicate important information about software to users."  Does that help? I tried to find a link to an article explaining badges but didn't find a good one - maybe this means we should have one in the FIT resources that could be linked in the form.
